index.html

```diff
@@ -1,301 +1,217 @@
-            <h1 class="my-name">I'm Yemsrach Glad to see you!</h1>
-           <p>I'm a software developer! I can help you build a product, feature or
-          website look though some of my work and experiance! if you like what you
-          see and have a project you need coded, don't hestiate to conduct me.
-        </p>
-        <div class="contact">
-          <img class="connect" src="./image/LET’S CONNECT.png" alt="connect">
-          <div class="contact_wraper">
-            <ul class="social_media">
-              <li class="social_media_list">
-                <a href="https://twitter.com/TamratYemsrach/">
-                  <img src="./image/twitter.svg" alt="twitter"
-                /></a>
-              </li>
-              <li class="social_media_list">
-                <a href="https://www.linkedin.com/in/yemsrach/"><img src="./image/Linkedin icon.svg" alt="Linkedin" /></a
-                >
-              </li class="social_media_list">
-              <li>
-                <a href="/"><img width="25px" src="./image/medium.svg" alt="medium" /></a>
-              </li>
-              <li class="social_media_list">
-                <a href="https://github.com/Yemisirach"
-                  ><img src="./image/github.svg" alt="github"
-                /></a>
-              </li>
-              <li  class="social_media_list" ><a href="#"><img class="vector" src="./image/Vector (1).png" alt="Vector"></a></li>
-            </ul>
-          </div>
-          </div>
+    <head>
+        <meta charset="UTF-8"/>
+        <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+        <title>Yemi's portfolio</title>
+        <link` rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"/>
+        <link rel="stylesheet" href="./styles.css"/>
+    </head>
+    <body>
+        <!-- header start -->
+        <header class="navbar_section">
+            <div class="navbar_logo">
+                <a href="#">YEMI</a>
+            </div>
+            <div class="dropdown_buttom">
+                <nav>
+                    <nav class="nav_botton"><img width="36px" height="34px" src="./image/habar.jpeg" alt="hambeger"></nav>
+                    <div class="drop_downbtn">
+                        <a href="#work">Portfolio</a>
+                        <a href="#About">About</a>
+                        <a href="#Contact">Contact</a>
+                    </div>
+                </nav>
+            </div>
+        </header>
+        <!-- header end -->
+        <!-- Home start -->
+        <section class="About">
+            <div class="ellipse">
+                <img class="ellipse_zero" src="./image/Ellipse 8.png" alt="ellipse">
+                <div class="ellipse_1">
+                    <img class="ellipse_zero_1" src="./image/Ellipse 5.png">
+                    <div class="ellipse_2">
```

Comment:
- [ ] You add the alternatives on your images almost everywhere :clap: , but I found that you forget to add on this image :squid: , Kindly make sure you add it here.
`<img class="ellipse_zero_1" src="./image/Ellipse 5.png" alt="image_alt">`